src/sandstorm/run-bundle.c++

```diff
@@ -2611,10 +2620,6 @@ private:
     KJ_SYSCALL(setenv("HTTP_GATEWAY", "local", true));
 
     KJ_SYSCALL(setenv("PORT", kj::str(config.ports[0]).cStr(), true));
-    KJ_IF_MAYBE(httpsPort, config.httpsPort) {
-      // TODO(cleanup): At this point, all this does is tell Sandcats to refresh certs.
```

Comment:
I'm fairly sure at this point this doesn't do *anything*; the sandcats logic doesn't look for this variable, and grepping around for it didn't come up with any uses.